### PR TITLE
Adding Patcher pricing page

### PR DESCRIPTION
**Pull request by ina-stoyanova**
Closes https://github.com/gruntwork-io/gruntwork-io.github.io/issues/544.

Preview: https://deploy-preview-552--keen-clarke-470db9.netlify.app/patcher/pricing/

Preview screenshot: 
Screenshot 2021-08-04 at 14 00 23

### CODE REVIEW

File: pages/landing/patcher/pricing/index.html [hunk 1]
```diff
@@ -1,72 +1,81 @@
 ---
 layout: landing
 type: landing
-slug: index-page
+slug: pricing
 
-title: Pricing
+title: Pick a plan
 excerpt: Configure the plan just right for your team.
```
Comment (robmorgan) [inline pages/landing/patcher/pricing/index.html:7]:
```suggestion
excerpt: Pick the plan just right for your team.
```
I suggest we change this heading as I feel our current pricing structure doesn't include optional add-ons, therefore to me it feels we are not 'configuring' a plan, but rather selecting one.

**Comment by ina-stoyanova** [comment]
Context: It has 3 tiers, based on the comment on the Pricing info on Notion for Patcher. It is a lot of information to include when the Basic version is free, so instead of complicating the Basic version with when it could be free, I've decided that we should have the 3rd option. So now we have 3 pricing tiers: "free, basic, enterprise". Updated the pricing based on a chat with @robmorgan and we followed the 50 USD guide price on dependabot.

File: pages/landing/patcher/pricing/index.html [hunk 2]
```diff
@@ -1,76 +1,86 @@
...
-        description: Description of Feature 3
-      - name: Feature 4
-        description: Description of Feature 4
 
-  - title: Option 2
-    description: Description of Option 2
-    price: "1,495"
-    image: /assets/img/gcp-logo@2x.png
+  - title: Pro
+    description: For teams operating up to 25 repos in a VSC organization account.
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/index.html:63]:
```suggestion
    description: For teams operating up to 25 repos in any organisation account.
```

File: pages/landing/patcher/pricing/index.html [hunk 3]
```diff
@@ -1,76 +1,86 @@
...
-      - name: Feature 3
-        description: Description of Feature 3
-      - name: Feature 4
-        description: Description of Feature 4
-      - name: Feature 5
-        description: Description of Feature 5
 
   - title: Enterprise
-    description: Description of Enterprise
+    description: All of Patcher's features and full control over the workflow for every repo in a VSC organization account.
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/index.html:69]:
```suggestion
    description: All of Patcher's features and full control over the workflow for every repo in a any organisation account.
```

File: pages/landing/patcher/pricing/_price-rows.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<tr>
+  <td style="width: 25%">&nbsp;</td>
+  {% for pricing in page.pricing %}
+    <td style="width: 25%; border-left: 1px solid #194c5f; border-right: 1px solid #194c5f; background-color: #252e3a;{% if include.bottom_border %} border-bottom: 1px solid #194c5f{% endif %}" class="text-center">
+      {% if pricing.price == "Contact Us" %}
+        <p><a class="btn btn-primary" href="{{ page.permalink }}contact/?selection={{ pricing.title | slugify }}" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Contact Us</a></p>
+        <p class="text-muted small">
+          <em>
+            Currently in private beta.<br>
+            Annual subscription.
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/_price-rows.html:26]:
```suggestion
```

File: pages/landing/patcher/pricing/_price-rows.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          </em>
+        </p>
+      {% elsif pricing.price == "Free" %}
+        <p><a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="{{ page.permalink }}contact/?selection={{ pricing.title | slugify }}" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Request Access</a></p>
+        <p class="text-muted small">
+          <em>
+            Currently in private beta.
+            Annual subscription.
+          </em>
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/_price-rows.html:35]:
```suggestion
```

File: pages/landing/patcher/pricing/_price-rows.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        </p>
+      {% else %}
+        <p><a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="{{ page.permalink }}contact/?selection={{ pricing.title | slugify }}" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Request Access</a></p>
+        <p class="text-muted small">
+          <em>
+            Currently in private beta.<br>
+            Annual subscription.
+          </em>
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/_price-rows.html:43]:
```suggestion
```

File: pages/landing/patcher/pricing/_price-rows.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  <td style="width: 25%">&nbsp;</td>
+  {% for pricing in page.pricing %}
+    <td style="width: 25%; border-left: 1px solid #194c5f; border-right: 1px solid #194c5f; background-color: #252e3a;{% if include.bottom_border %} border-bottom: 1px solid #194c5f{% endif %}" class="text-center">
+      {% if pricing.price == "Contact Us" %}
+        <p><a class="btn btn-primary" href="{{ page.permalink }}contact/?selection={{ pricing.title | slugify }}" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Contact Us</a></p>
+        <p class="text-muted small">
+          <em>
+            Currently in private beta.<br>
+            Annual subscription.
+          </em>
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/_price-rows.html:27]:
```suggestion
```

File: pages/landing/patcher/pricing/_price-rows.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  {% for pricing in page.pricing %}
+    <td style="width: 25%; border-left: 1px solid #194c5f; border-right: 1px solid #194c5f; background-color: #252e3a" class="text-center">
+      <hr>
+      {% if pricing.price == "Contact Us" %}
+        <p class="h1 enterprise-h1">Contact Us</p>
+      {% elsif pricing.price == "Free" %}
+        <p class="h1 enterprise-h1">Free</p>
+      {% else %}
+        <p class="big-price"><span>$</span>{{ pricing.price }}</p>
+        <p class="help-block" style="margin-top:-9px;">per repo / per month</p>
```
Comment (ina-stoyanova) [inline pages/landing/patcher/pricing/_price-rows.html:12]:
```suggestion
        <p class="help-block" style="margin-top:-9px;">per month</p>
```

**Review by infraredgirl — APPROVED** [review]

**Comment by ina-stoyanova** [comment]
Thanks for the approval, @infraredgirl! Merging now!

**Pull request merged**